openedx/adg/admin_dashboard/webinars/forms.py

```diff
@@ -0,0 +1,138 @@
+"""
+Forms for webinars app.
+"""
+import datetime
+import json
+
+import pytz
+from django import forms
+from django.conf import settings
+from django.contrib.auth.models import User
+
+from openedx.adg.common.lib.zoom_client.client import ZoomClient
+from openedx.adg.lms.webinars.constants import WEBINAR_DEFAULT_TIME_ZONE
+from openedx.adg.lms.webinars.models import Webinar
+
+from .constants import WEBINAR_OPTIONS
+
+
+class WebinarForm(forms.ModelForm):
+    """
+    Webinar Form to create/edit a webinar from admin dashboard
+    """
+
+    class Meta:
+        model = Webinar
+        fields = ['title', 'description', 'language', 'meeting_link', 'webinar_options',
+                  'start_time', 'duration', 'webinar_timezone',
+                  'audio_settings', 'enable_host_video', 'enable_panelist_video', 'banner',
+                  'invites_by_email_address', 'panelists']
+
+    def __init__(self, *args, **kwargs):
+        """
+        Override to initialize and clean the fields
+        """
+        super(WebinarForm, self).__init__(*args, **kwargs)
+        self.fields['meeting_link'].disabled = True
+
+        self.initialize_fields()
+        self.context = self.get_context_data()
+
+        if self.data:
+            self.data = self.data.copy()
+            self.data['webinar_options'] = self.format_webinar_options(self.data.getlist('webinar_options'))
+            self.data['start_time'] = self.format_start_time()
+            self.data['duration'] = datetime.timedelta(
+                hours=int(self.data.get('duration_hours', 0)), minutes=int(self.data.get('duration_mins', 0))
+            )
+            self.data.setlist(
+                'panelists', list(User.objects.filter(email__in=self.data.get('panelists', '').split(',')))
+            )
+
+    def format_webinar_options(self, user_selected_webinar_option):
+        """
+        Convert user selected webinar options into JSON format
+        """
```

Comment:
Let's please add description for `user_selected_webinar_option` in the doctring under Arguments and I think `user_selected_webinar_option` would contain a list therefore maybe we should rename it to `user_selected_webinar_options`